msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,348 @@
+* The sign-in screen area use must not use large logos or display an entire webpage.
+* The canvas can be simplified by breaking them into multiple tabs to reduce the complexity.
+* Personal app tabs should not have a duplicate header. Remove the logo from the iframe since the personal app framework already displays the app icon and app name.
 
-* Tabs must provide focused content and avoid needless UI elements. This usually refers to unnecessary nested or layered navigation, an extraneous or irrelevant UI next to the content, or any links that take the user to unrelated content. For example, the following tab view omits navigation menus and only showcases the main content:
+### 5.3 Navigation
 
-![SharePoint web view](../../../../assets/images/faq/web-sp.png)  
-![SharePoint tab view](../../../../assets/images/faq/tab-sp.png)
+* Tabs must not have more than three levels of navigation.
+* The secondary and tertiary pages in a tab must be opened in a level two and level three view in the main tab area that is navigated via breadcrumbs or left nav. You can also include these components to aid tab navigation:
+    * Back buttons
+    * Page headers
+    * Hamburger menus
+* Tab should not have horizontal scroll.
+* Deep links in tabs must not link to an external webpage but instead link somewhere within Teams (for example, task modules or other tabs).
 
-* Tabs must be light in nature and not include complex navigation.
-* Channel tabs that have complex editing capabilities within the app must open the editor view in a multi-window rather than a tab.
-* Channel tabs must not provide an app bar with icons in the left rail that conflicts with the main Teams navigation.
-* Tabs must not present an app bar with icons in the left rail that conflict with the main Teams navigation.
-* Tabs that have complex editing capabilities within the app must open the editor view in a multi-window rather than in the tab.
-* If there are multiple view options, consider having a tab config menu for the user to choose from. For example, instead of embedding a menu inside the tab, put the menu in the configuration page so the actual tab view is clean and focused.
-* Please include a *Help* tab as a static tab to advise users how to configure, sign-up, and use your app.
-* Please include a *Settings* tab that is available from the app header.
+### 5.4 Usability
 
-![Wide idea configuration page](../../../../assets/images/faq/wideidea.png)
+* Users must be able to undo their last action in the tab.
+* Personal app tabs may aggregate content from team or group chat scoped instances of that app in a single view.
+* Tabs must be responsive to Teams themes. When the Teams theme is changed, the theme within the app must also change to reflect that theme.
+* Personal app tabs must use Teams-styled components (adopting Teams fonts, type ramps, color palettes, grid system, motion, tone of voice, etc.) whenever possible.
```

Comment:
@heath-hamilton, Remove Personal app - 
 Tabs must use Teams-styled components (adopting Teams fonts, type ramps, color palettes, grid system, motion, tone of voice, etc.) whenever possible.